bin/s2nc.c

```diff
@@ -104,6 +106,8 @@ void usage()
                     "    accept: Accept all server requests for a new handshake\n"
                     "    reject: Reject all server requests for a new handshake\n"
                     "    wait: Wait for additional application data before accepting server requests. Intended for the integ tests.\n");
+    fprintf(stderr, "  --npn \n");
+    fprintf(stderr, "    Indicates support for the NPN extension. The alpn option MUST be used with this option to signal the protocols supported."); 
```

Comment:
Do you think this might be clearer?
```suggestion
    fprintf(stderr, "  --npn \n");
    fprintf(stderr, "    Indicates support for the NPN extension. The '--alpn' option MUST be used with this option to signal the protocols supported."); 
```